### PR TITLE
Added (initial) site configuration for the production

### PR DESCRIPTION
**Pull request by himorin**

### CODE REVIEW

**Comment by himorin** [comment]
check mail.notice_from

**Comment by hy0shida** [comment]
Do you think messages should appear to come from the subaru.nao.ac.jp domain? I think the Subaru Computer and Data Management Division doesn't in general like an idea of creating service (= non-human) accounts.

File: group_vars/site_pfs.sum.subaru.nao.ac.jp.yml [hunk 1]
```diff
@@ -0,0 +1,46 @@
+---
+site_config:
+  ntp: 
+    upstream: "ntp.sum.subaru.nao.ac.jp"
+    local: "ntp.pfs.sum.subaru.nao.ac.jp"
```
Comment (himorin) [inline group_vars/site_pfs.sum.subaru.nao.ac.jp.yml]: change to defined hostname (or define it...)

File: group_vars/site_pfs.sum.subaru.nao.ac.jp.yml [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+        home: "/var/lib/libvirt"
+        group: "libvirt-qemu"
+        groups: "kvm"
+      - uname: "postgres"
+        uid: 203
+        home: "/var/lib/postgres"
+        group: "postgres"
+        shell: "/bin/bash"
+  virt:
+    nfsdisk: "nfs-ics:/virt"
```
Comment (himorin) [inline group_vars/site_pfs.sum.subaru.nao.ac.jp.yml]: change to defined hostname (or define it...)

**Pull request merged**